scripts/code_generation/jsoncan/src/json_parsing/parse_tx.py

```diff
@@ -247,11 +248,9 @@ def _get_parsed_can_message(
             "telem_cycle_time", msg_cycle_time
         )
 
-    # Check if message ID is unique
-
     signals = []
     next_available_bit = 0
-    occupied_bits: list[Optional_t[str]] = [None] * 64
+    occupied_bits: list[Optional[str]] = [None] * max_len_bits
```

Comment:
optional here refers to the Schema type optional not the optional python type

Reply:
real shit